enterprise_subsidy/apps/api/v1/tests/mixins.py

```diff
@@ -75,21 +75,23 @@ def setUp(self):
         self.desired_system_wide_role = None
         self.desired_feature_role = None
 
-    def set_up_admin(self):
+    def set_up_admin(self, enterprise_uuids=None):
```

Comment:
I don't object to adding this parameter, but for this PR it's a no-op right?  Just trying to make sure I understand what's happening.

Reply:
Yeah, I think I'll want this for more test cases that assert access isn't allowed.  Currently it's not doing anything (besides making it slightly more obvious that `set_up_admin()` calls are associating the test case user with that exact enterprise uuid)